pkg/oc/cli/admin/policy/review.go

```diff
@@ -8,25 +8,24 @@ import (
 
 	"github.com/spf13/cobra"
 
+	appsv1 "k8s.io/api/apps/v1"
```

Comment:
It's always good to prefix these aliases with `k`, as in `kappsv1` so it's not mistaken by our apps.